utils/jsonapi.js

```diff
@@ -39,14 +38,6 @@ const serializerOptions = (serializerArgs) => {
     attributes: resourceKeys,
     id: identifierField,
     keyForAttribute: keyForAttribute || 'camelCase',
-    dataLinks: {
-      self: (row) => {
-        if (enableDataLinks) {
-          return resourcePathLink(resourceUrl, row[identifierField]);
-        }
-        return null;
-      },
-    },
```

Comment:
What was the reasoning for turning off resource-level self links? Usually we do this for all of our apis, so when a client does something  like `GET /parks` and finds a resource they're looking for, they know that they can access the resource at its self link. Also, if you want to turn these off, you would want to do this by calling `serializerOptions({ ..., enableDataLinks: false, ... });` so you don't have to modify `utils/jsonapi.js`.